helper/tests/helper-tests-allavro/src/test/java/com/linkedin/avroutil1/compatibility/AvroCompatibilityHelperParsingTest.java

```diff
@@ -232,4 +232,23 @@ public void testParsingHorribleLooseNumericDefaults() throws Exception {
       }
     }
   }
+
+  @Test
+  public void testParseTrailingContent() throws Exception {
```

Comment:
Maybe one more check to assert that the loose config continues to allow dangling content?